reporter/src/main/kotlin/ReportTableModel.kt

```diff
@@ -61,11 +59,6 @@ data class ReportTableModel(
      */
     val issueSummary: IssueTable,
 
-    /**
-     * A [SummaryTable] containing the dependencies of all [Project]s.
-     */
-    val summary: SummaryTable,
```

Comment:
As `ReportTableModel` is public, would this need to be marked as breaking API change again?